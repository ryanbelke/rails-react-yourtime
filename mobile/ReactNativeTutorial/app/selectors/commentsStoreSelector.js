export default state => state.get('commentsStore');
