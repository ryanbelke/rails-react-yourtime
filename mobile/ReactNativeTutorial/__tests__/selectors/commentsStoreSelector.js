import { fromJS } from 'immutable';
import commentsStoreSelector from 'ReactNativeTutorial/app/selectors/commentsStoreSelector';

describe('commentsStoreSelector', () => {
  it('selects the comments store', () => {
    const state = fromJS({
      commentsStore: {
        1: { id: 1, author: 'Alexey', text: 'Random comment' },
        2: { id: 2, author: 'Justin', text: 'Another random comment' },
      },
    });
    expect(commentsStoreSelector(state)).toMatchSnapshot();
  });
});
