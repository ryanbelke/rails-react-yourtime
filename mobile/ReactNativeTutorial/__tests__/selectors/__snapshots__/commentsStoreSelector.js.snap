exports[`commentsStoreSelector selects the comments store 1`] = `
Object {
  "1": Object {
    "author": "Alexey",
    "id": 1,
    "text": "Random comment",
  },
  "2": Object {
    "author": "Justin",
    "id": 2,
    "text": "Another random comment",
  },
}
`;
